### PR TITLE
Feature/code tweaks

### PR DESCRIPTION
**Pull request by kellenmace**
Rejigger a few little things in the functions & template tags files.

### CODE REVIEW

**Comment by gregrickaby** [comment]
@kellenmace You're PR contains PHP errors. Can you please fix and resubmit? Thanks!

**Comment by kellenmace** [comment]
@gregrickaby Thanks for the heads up! Pushed a fix and looks like all the Travis checks are passing now 👍🏼

File: inc/template-tags.php [hunk 1]
```diff
@@ -325,6 +320,39 @@ function _s_display_social_icons() {
 }
 
 /**
+ * Get the Twitter social sharing URL for the current page.
```
Comment (gregrickaby) [inline inc/template-tags.php]: @kellenmace Honest question, Do you think these getters should be moved to `extras.php` or maybe `queries.php`?

I'd like to keep `template-tags.php` as lean as possible...

**Comment by kellenmace** [comment]
@gregrickaby Good question. What do you think about moving the markup in `_s_display_social_icons()` into a `content-social-icons.php` template part like in my last commit? Since that function doesn't take any parameters and just immediately outputs markup, it seems like it's a perfect candidate to be moved into a template part. And if we do that, I think it makes sense to leave the functions for getting the share URLs in `template-tags.php`, since they themselves would be template tags. Whatchu think?

**Comment by gregrickaby** [comment]
@kellenmace Sounds good to me. Let me share this with the FEDs on today's scrum! Thanks for your PR dude! 👍

**Comment by kellenmace** [comment]
woo hoooo - thanks. Welcome!

**Review by gregrickaby — CHANGES_REQUESTED** [review]
@kellenmace After speaking with the group, can you rename the template part to: `sharing-icons.php`

**Review by gregrickaby — APPROVED** [review]

**Pull request merged**